thpool.c

```diff
@@ -53,11 +53,8 @@
 #define STRINGIFY(x) #x
 #define TOSTRING(x) STRINGIFY(x)
 
-static volatile int threads_keepalive;
 static volatile int threads_on_hold;
```

Comment:
I think that we are to fix this, we should also consider approach for `threads_on_hold`